src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CreateCaseCallbackTest.java

```diff
@@ -0,0 +1,127 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.controllers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.restassured.response.ValidatableResponse;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.config.IntegrationTest;
+
+import java.io.IOException;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.containing;
+import static com.github.tomakehurst.wiremock.client.WireMock.givenThat;
+import static com.github.tomakehurst.wiremock.client.WireMock.okJson;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.google.common.io.Resources.getResource;
+import static com.google.common.io.Resources.toByteArray;
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.text.MatchesPattern.matchesPattern;
+import static org.springframework.http.HttpStatus.OK;
+
+@IntegrationTest
+class CreateCaseCallbackTest {
+
+    private static final String CASE_ID_REGEX =
+        // will be replaced with real one after ccd api
+        "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$";
+
+    private static final String EVENT_ID = "createCase";
+    private static final String IDAM_TOKEN = "idam-token";
+    private static final String USER_ID = "user-id";
+
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+
+    @LocalServerPort
+    int serverPort;
+
+    @Test
+    void should_create_exception_record_if_classification_new_application_with_documents_and_ocr_data() {
```

Comment:
do you mean `should_create_service_case` ?

Reply:
good point. just copied/moved from unit tests. but yea - let me rename